compiler/rustc_hir_typeck/src/method/probe.rs

```diff
@@ -1981,6 +1987,46 @@ impl<'a, 'tcx> ProbeContext<'a, 'tcx> {
         }
     }
 
+    /// Determine if the associated item withe the given DefId matches
+    /// the desired name via a doc alias.
+    fn matches_by_doc_alias(&self, def_id: DefId) -> bool {
```

Comment:
This function has a lot of indentation. Can you try to reduce it?

For example:
```rust
let Some(name) = self.method_name else {return false};
```

```rust
if let Some(lit) = meta.name_value_literal()  && name.as_str() == lit.symbol.as_str() {
      return true;
}
```

Reply:
Good point, shaved id down a bit.